doc/migration/version-upgrade-guide.md

```diff
@@ -973,6 +977,31 @@ With:
 - name: {{your operator name which is the value of metadata.name in this file}}
 ```
 
+### Breaking Changes on Commands
+
+This release contains breaking changes in some commands.
+
+- The --namespace flag from `operator-sdk run --local` command, `operator-sdk test --local` command and `operator-sdk cleanup` command was deprecated and was replaced by --watch-namespace and --operator-namespace .
+
+Note that --watch-namespace can be used to set the namespace(s) which the operator will watch for changes. It will set the environment variable WATCH_NAMESPACE. Use explicitly an empty string to watch all namespaces or inform a List of namespaces such as "ns1,ns2" when the operator is cluster-scoped. If you use a List, then it needs contains the namespace where the operator is "deployed" in since the default metrics implementation will manage resources in the Operator's namespace. By default, it will be the Operator Namespace.
+
+Then, use the flag --operator-namespace to inform the namespace where the Operator will be "deployed" in and then, it will set the environment variable OPERATOR_NAMESPACE. If this value is not set, then it will be the namespace defined as default in the Kubeconfig.
+
+- Deprecated: The flag `--namespace` in `operator-sdk run --local` is deprecated. It will be removed in future versions.
+  The functionality will be supported by the new `--watch-namespace` flag.
+- `--watch-namespace` flag is added to `operator-sdk run --local` command. This flag works exactly like the `--namespace` flag.
+  This flag was added to align flag names with changes in `operator-sdk test --up-local` commnand.
+- Deprecated: The flag `--namespace` in `operator-sdk test --up-local` is deprecated. It will be removed in future versions.
+  The functionality will be supported by the new `--operator-namespace` flag.
+- `--operator-namespace` is added to `operator-sdk test --up-local` command. This flag specifies the namespace in which Namespaced manifests
+  are installed.
+- `--watch-namespace` is added to `operator-sdk test --up-local` command. This flag specifies the namespace in where the operator watches
+   for changes. This flag can be set to "foobar", "" (watch-all-namespace) or left unset (when not set watch-namespace = operator-namespace)*[]:
+- The `ctx.GetNamespace()` in `pkg/test` is replaced with `ctx.GetOperatorNamespace()` and `ctx.GetWatchNamespace()` to accomodate the
+  changes in flags.
```

Comment:
I think we need to remove line 990 to 1001.
It is already described above.

Reply:
done